tests/snappi_tests/multidut/pfc/test_lossless_response_to_throttling_pause_storms.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+import random
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.fixtures.conn_graph_facts import conn_graph_facts, \
+    fanout_graph_facts                                                                          # noqa: F401
+from tests.common.snappi_tests.snappi_fixtures import snappi_api_serv_ip, snappi_api_serv_port, \
+    snappi_api, snappi_dut_base_config, get_tgen_peer_ports, get_multidut_snappi_ports, \
+    get_multidut_tgen_peer_port_set, cleanup_config                                             # noqa: F401
+from tests.common.snappi_tests.qos_fixtures import prio_dscp_map, \
+    lossless_prio_list                                                                          # noqa: F401
+from tests.snappi_tests.variables import config_set, line_card_choice
+from tests.snappi_tests.multidut.pfc.files.lossless_response_to_throttling_pause_storms_helper import (
+    run_lossless_response_to_throttling_pause_storms_test)
+from tests.common.snappi_tests.snappi_test_params import SnappiTestParams
+
+pytestmark = [pytest.mark.topology('multidut-tgen')]
+
+
+@pytest.mark.parametrize('line_card_choice', [line_card_choice])
+@pytest.mark.parametrize('linecard_configuration_set', [config_set])
+def test_lossless_response_to_throttling_pause_storms(snappi_api,                       # noqa: F811
+                                                      conn_graph_facts,                 # noqa: F811
+                                                      fanout_graph_facts,                # noqa: F811
+                                                      line_card_choice,
+                                                      duthosts,
+                                                      prio_dscp_map,                    # noqa: F811
+                                                      lossless_prio_list,               # noqa: F811
+                                                      linecard_configuration_set,       # noqa: F811
+                                                      get_multidut_snappi_ports,):      # noqa: F811
+
+    """
+    Run PFC watchdog test under many to one traffic pattern
+
+    Args:
+        snappi_api (pytest fixture): SNAPPI session
+        snappi_testbed_config (pytest fixture): testbed configuration information
+        conn_graph_facts (pytest fixture): connection graph
+        fanout_graph_facts (pytest fixture): fanout graph
+        duthosts (pytest fixture): list of DUTs
+        lossy_prio_list (pytest fixture): list of lossy priorities
+        prio_dscp_map (pytest fixture): priority vs. DSCP map (key = priority)
+        line_card_choice: Line card choice to be mentioned in the variable.py file
+        linecard_configuration_set : Line card classification, (min 1 or max 2  hostnames and asics to be given)
+
```

Comment:
Do we expect loss to happen or just traffic reduction ? I believe it should just reduce traffic and no loss for lossless traffic.